src/core/stream.c

```diff
@@ -44,6 +44,36 @@ static struct {
 	    .listener_alloc = nni_ipc_listener_alloc,
 	},
 #endif
```

Comment:
To make NNG MQTT compatible is our goal, I don't think it will mislead users to use SP with MQTT, as long as we provide clear and straight docs. However, we can make a totally independent URL translator and dialer/initiator for MQTT, but needs your hands and help to do it.